.changeset/poor-steaks-remain.md

```diff
@@ -0,0 +1,7 @@
+---
+'@backstage/plugin-permission': patch
```

Comment:
@freben For adding new plugins, should this be a minor instead?

Reply:
Honestly I think that would look nicer. Then the first release would be `0.1.0` which feels more appropriate than `0.0.1` :)

Reply:
(for no specific reason, really...)